### DISCUSSION TITLE
I build vscodium in win11 but after built open the codium.exe it show nothing, anyone face this problem?thanks so much

### DISCUSSION THREAD

**Discussion by zhilee2008** [VSCodium/vscodium#2111] [Q&A]
![image](https://github.com/user-attachments/assets/01091b35-9411-4f7a-a03a-562e70609d16)
![image](https://github.com/user-attachments/assets/19e3bcc0-32e6-405a-ab14-3be63eda1cf5)

**Comment by zhilee2008** [comment]
Found the root cause
because I built vsdium
 `VSCode-win32-x64\resources\app\out\vs\code\electron-sandbox\workbench\workbench.js`  is different from release version
but how to fix this?anyone has anyidea?

**Comment by daiyam** [comment]
Yep, since `1.95`, I can't build VSCodium directly on Windows. I didn't figure out the issue...

But it seems you have a clue!

**Reply by zhilee2008** [reply]
Yes,So I just know the root cause but I havn't found a solution for this issue.
